public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -333,6 +333,9 @@ define(function(require) {
       $("#navbar-publish-button").click(showPublishDialog);
       $("#publish-button-cancel").click(hidePublishDialog);
 
+      //Publish link
+      $("#link-publish-link").click(hidePublishDialog);
```

Comment:
It actually doesn't, just like the Publish button or any button that uses the click() function.